docs/Home/core-concepts.md

```diff
@@ -0,0 +1,17 @@
+# Messages, Channels & Schemas
```

Comment:
These terms should be defined in the specification. If they are, should these sections just link to the relevant definition in the spec?